src/pages/editor/curators/AddCuratorForm.tsx

```diff
@@ -162,14 +162,17 @@ export const AddCuratorForm = ({
 
               if (isRemovingRole) {
                 // Remove roles that depend on the removed role
-                if (role === RoleName.PublishingCurator) {
-                  newRoles = newRoles.filter(
-                    (role) =>
-                      role.rolename !== RoleName.CuratorThesis && role.rolename !== RoleName.CuratorThesisEmbargo
-                  );
-                } else if (role === RoleName.CuratorThesis) {
+                if (role === RoleName.CuratorThesis) {
                   newRoles = newRoles.filter((role) => role.rolename !== RoleName.CuratorThesisEmbargo);
                 }
+              } else {
+                if (role === RoleName.CuratorThesisEmbargo) {
```

Comment:
Kunne spart ett innrykk ved å løfte opp sjekken. Men syns egentlig det er mer lesbart som det står nå uansett da😅 

```suggestion
              } else if (role === RoleName.CuratorThesisEmbargo) {
```

Reply:
ja, var det som var tanken min også